client/my-sites/stats/stats-purchase/stats-purchase-notice.jsx

```diff
@@ -27,44 +25,68 @@ const StatsCommercialOwned = ( { siteSlug } ) => {
 				) }
 			</p>
-			<div className={ `${ COMPONENT_CLASS_NAME }__benefits` }>
-				<ul className={ `${ COMPONENT_CLASS_NAME }__benefits--included` }>
-					<li>{ translate( 'Real-time data on visitors' ) }</li>
-					<li>{ translate( 'Traffic stats and trends for post and pages' ) }</li>
-					<li>{ translate( 'Detailed statistics about links leading to your site' ) }</li>
-					<li>{ translate( 'GDPR compliant' ) }</li>
-					<li>{ translate( 'Access to upcoming advanced features' ) }</li>
-					<li>{ translate( 'Priority support' ) }</li>
-					<li>{ translate( 'Commercial use' ) }</li>
-				</ul>
-			</div>
+			<StatsBenefits />
 			<Button variant="primary" onClick={ handleClick }>
 				{ translate( 'See your stats' ) }
 			</Button>
 		</>
 	);
 };
 
+const StatsPWYWOwned = () => {
+	const translate = useTranslate();
+
+	return (
+		<>
+			<div className="stats-purchase-page__notice--with-banner">
+				<NoticeBanner
+					level="success"
+					title={ translate( 'Thank you for using Jetpack Stats!' ) }
+					hideCloseButton
+				>
+					{ translate(
+						'You have already purchased a personal plan. If you want to upgrade to a commercial plan you can do it below.'
+					) }
+				</NoticeBanner>
+			</div>
+		</>
+	);
+};
+
+const StatsFreeOwned = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="stats-purchase-page__notice--with-banner">
+			<NoticeBanner
+				level="success"
+				title={ translate( 'Thank you for using Jetpack Stats!' ) }
+				hideCloseButton
+			>
+				{ translate(
+					'We appreciate your continued support. If you want to access upcoming features, please consider upgrading.'
+				) }
+			</NoticeBanner>
+		</div>
+	);
+};
+
 const StatsPurchaseNotice = ( { siteSlug } ) => {
-	// TODO: use props to swap multiple versions of the left side content
+	return (
+		<StatsSingleItemPagePurchaseFrame>
+			<StatsCommercialOwned siteSlug={ siteSlug } />
+		</StatsSingleItemPagePurchaseFrame>
+	);
+};
 
+const StatsPurchaseNoticePage = ( { siteSlug, isCommercialOwned, isFreeOwned, isPWYWOwned } ) => {
 	return (
-		<div className={ classNames( COMPONENT_CLASS_NAME, `${ COMPONENT_CLASS_NAME }--single` ) }>
-			<Card className={ `${ COMPONENT_CLASS_NAME }__card-parent` }>
-				<div className={ `${ COMPONENT_CLASS_NAME }__card` }>
-					<div className={ `${ COMPONENT_CLASS_NAME }__card-inner--left` }>
-						<StatsCommercialOwned siteSlug={ siteSlug } />
-					</div>
-					<div className={ `${ COMPONENT_CLASS_NAME }__card-inner--right` }>
-						<StatsPurchaseSVG />
-						<div className={ `${ COMPONENT_CLASS_NAME }__card-inner--right-background` }>
-							<img src={ statsPurchaseBackgroundSVG } alt="Blurred background" />
-						</div>
-					</div>
-				</div>
-			</Card>
+		<div className="stats-purchase-page__notice">
+			{ isCommercialOwned && <StatsPurchaseNotice siteSlug={ siteSlug } /> }
```

Comment:
We may need to apply the `StatsNotices` [system](https://github.com/Automattic/wp-calypso/blob/add/stats-80259-commercial-flow/client/my-sites/stats/stats-notices/index.tsx#L28-L41) to avoid showing all the notices on the same page.